corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -111,6 +111,20 @@ hqDefine("cloudcare/js/form_entry/form_ui", function () {
         }
     }
 
+    function getMatchingStyles(pattern, styleStr) {
+        let retVal = [];
+        if (styleStr) {
+            let styles = styleStr.split(' ');
+            styles.forEach(function (style) {
+                if ((pattern instanceof RegExp && style.match(pattern))
```

Comment:
I think it would be enough to check the type of `pattern` once at the beginning of this function. And just return an empty list then.

Reply:
hm, i'm not following what you mean. I think the pattern type check determines how to match the pattern, so i think it needs to be done like how it is
```
if ((pattern instanceof RegExp && style.match(pattern))
      || (typeof pattern === "string" && pattern === style)
```

Reply:
Oh. I misread the if statement.